rules/apps.libsonnet

```diff
@@ -64,7 +68,7 @@
               sum by (namespace) (
                   sum by (namespace, pod) (
                       max by (namespace, pod, container) (
-                          kube_pod_container_resource_requests_memory_bytes{%(kubeStateMetricsSelector)s}
+                          kube_pod_container_resource_requests{%(kubeStateMetricsSelector)s, %(memorySelector)s, %(byteSelector)s}
```

Comment:
Any reason why we don't just include the `memorySelector` and `byteSelector` directly in this query? We try to keep configuration minimal if there aren't really any other ways to configure something.

Reply:
We can. I thought this was the preferred way of doing it. I'll change that.